src/api/app/components/patchinfo_component.html.haml

```diff
@@ -0,0 +1,71 @@
+.pb-3
+  %h4
+    = link_to(@path) do
+      = @patchinfo['summary']
+    = category
+    = rating
+    = stopped
+    = retracted
+  %p
+    Patch created by
+    - if (packager = User.find_by_login(@patchinfo['packager']))
+      = render UserAvatarComponent.new(packager.login)
+    - else
+      = @patchinfo['packager']
+    - if @patchinfo.key?('incident')
+      in the maintenance incident request
+      - if (incident = BsRequest.find_by(number: @patchinfo['incident']))
+        = link_to(request_show_path(incident)) do
+          \##{@patchinfo['incident']}
+      - else
+        \##{@patchinfo['incident']}
+
+  %h5 Patch description
```

Comment:
If there is no patch description, I suggest not displaying this heading and the `No description specified.` text at all. There's already possibly a lot of information included in a patchinfo, let's not make it harder to process with such text.

Reply:
Displaying nothing, so an empty description, doesn't bring any value to the user. I still suggest hiding it in this case.